src/views/Player.vue

```diff
@@ -206,6 +207,8 @@ export default defineComponent({
       sessionId: "", // id of the session created for a user-quiz combination
       isSessionAnswerRequestProcessing: false, // whether session answer api request is processing
       continueAfterAnswerSubmit: true as boolean, // do we continue after submitting answer
+      timeSpentPerQuestion: [] as number[], // stores time spent per each question
```

Comment:
Comparing with nomenclature of others -- this should just be `timeSpent` no need of `perQuestion` 

Or maybe rename to `timeSpentOnQuestion` -- that fits better IMO since here, an array means it's question-wise already, so the `per` doesn't make sense

Reply:
changed it